code/scenes/src/main/kotlin/org/fbme/scenes/cells/button/Button.kt

```diff
@@ -0,0 +1,9 @@
+package org.fbme.scenes.cells.button
+
+import java.awt.Graphics2D
+
+interface Button {
+    val height: Int
+    val width: Int
+    fun paint(g: Graphics2D, x: Int, y: Int, isDark: Boolean)
```

Comment:
I think more bullet-proof solution would load and render icons from csv. Idea has support for operating icon resources. Have you considered such solution?

Reply:
Added icons